draft-ietf-quic-tls.md

```diff
@@ -465,10 +465,9 @@ and lower encryption levels than the current encryption level used by TLS.
 
 In particular, server implementations need to be able to read packets at the
 Handshake encryption level before the final TLS handshake message at the 0-RTT
```

Comment:
What's the final TLS handshake message at the 0-RTT encryption level?  Overall, I'm not sure what this paragraph is trying to add to the previous one.